app/views/pages/index.html.haml

```
@@ -25,11 +25,11 @@
 .hero
   %h2 Discover and meet
   %p
-    = pluralize(@students_count, 'student')
+    = pluralize(@membership_count['StudentMembership'], 'student')
     and
-    = pluralize(@coaches_count, 'coach')
+    = pluralize(@membership_count['CoachMembership'], 'coach')
```

This is a hash, right? I'd prefer we use `fetch` here and above so that we fail hard and early in case the value shouldn't be there :)

---

@PragTob I have updated the PR. Kindly check if that's what you meant. Now I know the basic difference between .fetch() and [] on hashes, thanks for that. I later realized that .fetch() can take a second parameter which would be a default value instead of raising an error, hence reason for specifying 0 as the default value.